src/main.cpp

```diff
@@ -564,7 +564,27 @@ bool __fastcall LevelCell_onViewProfile(LevelCell* self, void* a, CCObject* b) {
 
     return true;
 }
-
+class LevelCel : public LevelCell
+{
+public:
+    void CopyID(CCObject* e)
+    {
+        // Yeah i took this from stack overflow
+        std::string s = std::to_string(this->level->levelID);
+        char const* pchar = s.c_str();
+        BetterInfo::copyToClipboard(pchar);
+        FLAlertLayer::create(nullptr, "Copied ID", "OK", nullptr, "Copied the levels <cj>ID</c> to your <cy>clipboard</c>.")->show();
```

Comment:
wouldnt it be better to use the copyToClipboard variant that displays the tooltip